content/general/concepts/rest/rest.md

```diff
@@ -0,0 +1,31 @@
+---
+Title: 'REST'
+Description: 'REST is an architectural style for API design that is defined by 6 constraints.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Development'
+Tags:
+  - 'APIs'
+  - 'Developer Tools'
+  - 'Dependency'
+  - 'Interface'
+CatalogContent:
+  - 'paths/computer-science'
+  - 'paths/front-end-engineer-career-path'
+---
+
+**REST** is an architectural style for [API](https://www.codecademy.com/resources/docs/general/api) design that is defined by six constraints. These constraints were first documented in Roy Fielding's dissertation ["Architectural Styles and the Design of Network-based Architectures"](https://www.ics.uci.edu/~fielding/pubs/dissertation/top.htm). The acronym REST stands for REpresentational State Transfer, which is foundational to this design architecture, the sharing of resources, or rather representations of those resources, across systems.
+
+The constraints that define REST architecture include the following:
```

Comment:
```suggestion
## REST Constraints
```

Could we change this to a header, feel free to adjust~

Reply:
> Could we change this to a header, feel free to adjust~

All set!